src/components/molecules/MarketStats.tsx

```diff
@@ -19,9 +22,30 @@ interface MarketStatsResponse {
 
 const refreshInterval = 60000 // 60 sec.
 
+const getTotalPoolsValues = gql`
+  query PoolsData {
+    poolFactories {
+      totalLockedValue
```

Comment:
This should be `totalValueLocked` (TVL) not `totalLockedValue`. That's the industry standard. If we don't follow the standard we will confuse people. Example: see "TVL" in https://defipulse.com/

Reply:
This a sub-graph issue, not here